mcs/packages/Makefile

```diff
@@ -3,7 +3,7 @@ include ../build/rules.make
 
 ROSLYN_CSC_DIR = $(dir $(CSC_LOCATION))
 
-ROSLYN_FILES = \
+ROSLYN_FILES_FOR_MONO = \
```

Comment:
Why? Most of the files are used from lib/mono/4.5 anyway, secondly the duplicated files are missing AOTed files making csc much slower than it has to be.